Okta.AspNetCore.WebApi.IntegrationTest/appsettings.json

```diff
@@ -1,15 +1,10 @@
-﻿{
+{
   "Logging": {
-    "IncludeScopes": false,
-    "Debug": {
-      "LogLevel": {
-        "Default": "Warning"
-      }
-    },
-    "Console": {
-      "LogLevel": {
-        "Default": "Warning"
-      }
+    "LogLevel": {
+      "Default": "Information",
+      "Microsoft": "Warning",
+      "Microsoft.Hosting.Lifetime": "Information"
     }
-  }
+  },
+  "AllowedHosts": "*"
```

Comment:
I'm not very familiar with the `AllowedHosts` config setting, but I was [reading here](https://docs.microsoft.com/en-us/aspnet/core/fundamentals/servers/kestrel?view=aspnetcore-3.1#endpoint-configuration) it the docs made it seem like omitting it altogether would prevent the host filtering middleware from even running. I was wondering if this would have the same effect as specifying `*` for the host filter.

Would it be better to omit this setting or is it needed for some reason?

Reply:
I believe we can omit this setting in this case. I'll double check.